plugins/kubernetes/test/models/kubernetes/resource_test.rb

```diff
@@ -805,5 +806,18 @@ def deployment_stub(replica_count)
         end
       end
     end
+
+    describe "#revert" do
+      it "deletes and then creates without resourceVersion because that is not allowed" do
+        with = ->(request) { request.body.wont_include "resourceVersion"; true }
```

Comment:
I find this test case rather hard to understand.. not sure if it's the variable names or the nesting of random requests